### PR TITLE
Add show missions and rockets booked method on my profile route

### PR DESCRIPTION
**Pull request by hienphan0111**
Compose two/three column layout and list ONLY the rockets reserved and missions joined by the user (as per design):

- [ ] Render a list of all joined missions (use filter()).

- [ ] Render a list of all reserved rockets (use filter()).

### CODE REVIEW

File: src/components/Mission.component.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ function Mission({ mission }) {
 
   const joinHandler = (e) => {
     const { id } = e.target;
+    console.log(id);
```
Comment (jodavid444) [inline src/components/Mission.component.jsx:18]: According JS best practice, kindly remove all console.log in your code to make it clean

Comment (jodavid444) [inline src/components/Mission.component.jsx]: According to JS best practice, kindly remove all console.log in your code to make it clean"

**Review by jodavid444 — CHANGES_REQUESTED** [review]
# Status: Changes Required  🔴

Hey @hienphan011 👋

Highlights 🥇
✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on for your project to be approved,  but you are almost there! 😊

Optional
All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration.

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/components/Mission.component.jsx (hunk 1 above)
Comment (hienphan0111) [reply src/components/Mission.component.jsx]: Thanks @jodavid444 ,
I already fixed it. Please review it again.

**Review by jodavid444 — APPROVED** [review]
Hi @hienphan0111  ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉





## Highlights
- No linter errors ✔️
-  Good code organization and working app ✔️
 - Correct Gitflow ✔️
 - Descriptive PR ✔️
 - Descriptive commit messages ✔️
 
Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**